internal/report/prometheus.go

```diff
@@ -11,77 +11,39 @@ const namespace = "popeye"
 var (
 	score = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: namespace,
-		Name:      "score",
-		Help:      "Score of kubernetes cluster.",
-	},
-		[]string{
-			"cluster",
-			"namespace",
-		})
-	grade = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: namespace,
-		Name:      "grade",
-		Help:      "Grade of kubernetes cluster. (1: A, 2: B, 3: C, 4: D, 5: E, 6: F)",
+		Name:      "cluster_score_total",
+		Help:      "Popeye's sanitizers overall cluster score.",
 	},
 		[]string{
 			"cluster",
 			"namespace",
 			"grade",
 		})
-	sanitizersOk = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: namespace,
-		Name:      "sanitizers_ok",
-		Help:      "Sanitizer ok level results for resource groups.",
-	},
-		[]string{
-			"cluster",
-			"namespace",
-			"title",
-		})
-	sanitizersInfo = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: namespace,
-		Name:      "sanitizers_info",
-		Help:      "Sanitizer info level results for resource groups.",
-	},
-		[]string{
-			"cluster",
-			"namespace",
-			"title",
-		})
-	sanitizersWarning = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: namespace,
-		Name:      "sanitizers_warning",
-		Help:      "Sanitizer warning level results for resource groups.",
-	},
-		[]string{
-			"cluster",
-			"namespace",
-			"title",
-		})
-	sanitizersError = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	sanitizers = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: namespace,
-		Name:      "sanitizers_error",
-		Help:      "Sanitizer error level results for resource groups.",
+		Name:      "sanitizers_findings_total",
+		Help:      "Popeye's sanitizers findings for resource groups.",
```

Comment:
@eminugurkenar How about `sanitizer_results_total` and `Popeye's sanitizer results for resource group.` As it describes a single sanitizer for a given resource?